packages/@okta/vuepress-site/code/javascript/okta_auth_sdk/index.md

```diff
@@ -23,8 +23,8 @@ If you'd like to explore the entire Auth SDK, please see the [Okta AuthJS Source
 You will need the following things for this guide:
 
 - An Okta org - If you don't have an existing org, register for [Okta Developer Edition](https://developer.okta.com/signup/).
-- An OpenID Connect Application. Instructions for creating one can be found on [this page](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_Apps_App_Integration_Wizard), under the "OpenID Connect Wizard" section.
-- At least one User [assigned to the Application](https://help.okta.com/en/prod/Content/Topics/Apps/Apps_Apps_Page.htm#Assigning).
+- An OpenID Connect Application. Instructions for creating one can be found on [this page](https://help.okta.com/en/prod/okta_help_CSH.htm#csh-attribute-statements-saml), under the "OpenID Connect Wizard" section.
```

Comment:
This is not the correct place to link to any more - the instructions for creating an OIDC app with the wizard are at:
https://help.okta.com/en/prod/Content/Topics/Apps/Apps_App_Integration_Wizard_OIDC.htm

Text should be changed to:
`- An OpenID Connect application. Instructions for creating one can be found on [this page](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_Apps_App_Integration_Wizard-oidc)`